01-Login/README.md

```diff
@@ -42,6 +42,13 @@ Open a terminal, go to the project root directory and run the following command:
 
 The server will be accessible on https://localhost:8080/portal/home. After logging in you should see the `token` in the header.
 
+### Running the sample with docker
+
+In order to run the example with docker you need to have `docker` installed.
```

Comment:
I'd add a link to the [installation guide](https://docs.docker.com/engine/installation/)